backend/extensions/Squidex.Extensions/Actions/SignalR/SignalRAction.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
```

Comment:
Please check if there are warnings from roslyn, e.g. header is missing and I am not sure if all usings are needed.

Reply:
it's corrected for header, all using are useful